.circleci/config.yml

```diff
@@ -290,6 +310,7 @@ jobs:
       - store_artifacts:
           path: _build/dev3/rel/epoch/log
           destination: node3/
+      - *fail_notification
```

Comment:
In order to finish ["Notify a dedicated chat channel for CI failures on master"](https://www.pivotaltracker.com/story/show/156197975) I expect all CI failure on master to be notified in chat channel - not only for job uat_tests.

Reply:
I really don't see an easy way to do that without splitting (duplicating) all the jobs to be master specific as the fail_notification is run as a job step (no builtin CCI notification I can found for v2.0).

Thus I kept this only for uat_tests as it's the only job that runs only on master. Other jobs all run on PR builds too, so the probability to fail when merged is low IMO.


Let me know if you have better idea how to achieve your request.

Reply:
@dincho Have you considered making notification conditional based on CIRCLE_BRANCH env var?

Reply:
@lucafavatella I just come up with the same idea on the terrace 🚬